src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -2535,6 +2522,7 @@ FixedwingPositionControl::tecs_update_pitch_throttle(const float control_interva
 	if (_vehicle_status.is_vtol) {
 		if (_vehicle_status.vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING || _vehicle_status.in_transition_mode) {
 			_tecs_is_running = false;
+			return;
```

Comment:
Is this ok? When you transition, it still is rotary wing until it finished transitioning, right? So the code below is not executed and this ramp up is not properly executed (although i would like to get rid of this anyways).

Reply:
Yes correct. I'll move the return.
And see https://github.com/PX4/PX4-Autopilot/pull/22510 - there I propose to remove the whole block.

Reply:
Fix: f1588db1caf6dbe6895ebd6d20ee84ef87aa98e4